swagger-templates/python/api.mustache

```diff
@@ -149,14 +131,14 @@ class {{classname}}(object):
 
         query_params = {}
 {{#queryParams}}
-        if '{{paramName}}' in params:
+        if '{{paramName}}' in params and params['{{paramName}}'] is not None:
```

Comment:
The `cursor` would be `None` in our api call and it is possible that the user put the None for cursor param so I added here.